index.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     'notice/notice': [
       'error',
       {
+        mustMatch: "Copyright %YEAR% Cortex Applications, Inc.".replace("%YEAR%", new Date().getFullYear().toString()),
```

Comment:
Ah actually I think I see why we don't want string interpolation so ignore part 1

Reply:
Well, I'm not sure about this file (and some others in this repo), because they have Spotify copyright and not Cortex. Should it be Cortex?

Reply:
I _think_ so but cc @nikhilunni for confirmation

Reply:
since this is a fork of spotify's settings with an apache license we can't remove the original copyright -- i think the proper thing to do is to add like a "Modified [year] [your name or company." in addition to their original copyright

Reply:
obviously not legal advice haha, but it's not a substantial change i don't think it's necessary to add our own

Reply:
Oh I missed that this was a fork of theirs 🙃